src/common/MaintainAspectRatio.tsx

```diff
@@ -10,12 +10,11 @@ export const MaintainAspectRatio = ({
     width: number;
     children: React.ReactNode;
 }): JSX.Element => (
-    /* https://css-tricks.com/aspect-ratio-boxes/ */
     <div
         css={css`
             /* position relative to contain the absolutely positioned iframe plus any Overlay image */
             position: relative;
-            padding-bottom: ${(height / width) * 100}%;
+            aspect-ratio: ${width / height};
```

Comment:
Have you checked the support for [`aspect-ratio`](https://caniuse.com/mdn-css_properties_aspect-ratio)?

Is this really necessary? If you are not removing the relative positioning, I would keep things as they were.

Reply:
Good call checking the support for aspect-ratio. 
Using `padding-bottom` doesn't work well with the IMA integration since the IMA script changes the [`ima-ad-container`](https://github.com/guardian/atoms-rendering/blob/9266a44016e6fcc74f74d87ec7a908d0c71b2976/src/ImaAdContainer.tsx#L8) styles directly, making it `position: relative` `display: block` and giving it pixel value width and height.
That results in the MaintainAspectRatio component actually appear to have a player-sized padding bottom while the ad is playing.
![Screenshot 2022-09-13 at 12 51 11](https://user-images.githubusercontent.com/17057932/189894815-35082885-0936-4a33-b158-0c03e6d41175.png)

`aspect-ratio` seemed to be a good way to fix this style issue without having to override the IMA script's styles while also improving readability.

Reply:
Overriding IMA styles like this seems to work as an alternative:
https://github.com/guardian/atoms-rendering/pull/447/commits/f3fa5634a03ed06e5878f25de9f5e9d0109cbd24

Reply:
I prefer the override of IMA styles, but you have also made a compelling argument for `aspect-ratio`. I’ll leave it up to you.

Reply:
I'm happy to stick with padding-bottom in the interest of keeping the changes to users who aren't in the 0% test as light as possible and since as you point out there are quite a few [browsers that don't support aspect ratio](https://caniuse.com/mdn-css_properties_aspect-ratio) in the [list of browsers we officially support](https://www.theguardian.com/help/recommended-browsers).
We can always reconsider when IMA ads are a bit more established.